File: ScoreSystem/ScoreSystem/ScoreSystem/Model/CriticalStudentLog.cs
using Org.BouncyCastle.Bcpg.Sig;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ScoreSystem.Model
{
    public class CriticalStudentLog
    {
        private long id;
        private long examId;
        private string studentNumber;
        private string studentName;
        private int universityLevel;
        private int scoreRank;
        private int targetRank;
        private int gap;
        private double score;
        private long subjectGroupId;

        public long Id { get => id; set => id = value; }
        public long ExamId { get => examId; set => examId = value; }
        public string StudentNumber { get => studentNumber; set => studentNumber = value; }
        public string StudentName { get => studentName; set => studentName = value; }
        public int UniversityLevel { get => universityLevel; set => universityLevel = value; }
        public int ScoreRank { get => scoreRank; set => scoreRank = value; }
        public int TargetRank { get => targetRank; set => targetRank = value; }
        public int Gap { get => gap; set => gap = value; }
        public double Score { get => score; set => score = value; }
        public long SubjectGroupId { get => subjectGroupId; set => subjectGroupId = value; }
    }
}

File: ScoreSystemServer/ScoreSystemServer/src/main/java/com/score/system/entity/school/CriticalStudentLog.java
package com.score.system.entity.school;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import com.score.system.entity.BaseEntity;
import lombok.Data;

@Data
@TableName("critical_student_log")
public class CriticalStudentLog extends BaseEntity {
    @TableId(type = IdType.AUTO)
    private Long id;
    @TableField("exam_id")
    private Long examId;
    @TableField("student_number")
    private String studentNumber;
    @TableField("student_name")
    private String studentName;
    @TableField("university_level")
    private Integer universityLevel;
    @TableField("score_rank")
    private Integer scoreRank;
    @TableField("target_rank")
    private Integer targetRank;
    private Integer gap;
    private Double score;
    @TableField("subject_group_id")
    private Long subjectGroupId;
}
